packages/strategies/stress.go

```diff
@@ -153,6 +325,14 @@ func (s *StressTest) ValidateExecution(ctx context.Context, e *eth.ExecutionPayl
 
 func (s *StressTest) ValidateBlock(ctx context.Context, e *eth.ExecutionPayloadEnvelope, a BlockCreationParams) error {
 	// Todo: Validate enough of the txns were included
+	includedTxCount := len(e.ExecutionPayload.Transactions)
+	exceptedTxCount := len(e.ExecutionPayload.Transactions)
+	blockNum := uint64(e.ExecutionPayload.BlockNumber)
+	s.logger.Info("included tx count", "blockNum", blockNum, "excepted", exceptedTxCount, "actually", includedTxCount)
+
+	if erc20Address == (common.Address{}) {
+		s.notifyCh <- struct{}{}
+	}
```

Comment:
Seems like we can probably revert this change. `includedTxCount == exceptedTxCount` in all cases as this is written. And I think we can rid of the `if` block if you agree `s.notifyCh` is not needed based on comment above

Reply:
It's done